test/regression/regression_test.cpp

```diff
@@ -130,7 +130,13 @@ TEST(RegressionTest, StateRecovery) {
 
   {
     integration_framework::IntegrationTestFramework(
-        1, dbname, [](auto &) {}, false, path)
+        1,
+        dbname,
+        [](auto &) {},
+        false,
+        std::chrono::milliseconds(20000),
```

Comment:
Maybe move waiting parameters after the block store path, so we can omit them here?